styles.css

```diff
@@ -78,4 +78,154 @@ body {
 
 ul {
   padding: 0;
+  list-style-type: none;
+}
+
+.section1 {
+  background-image: url('images/Snapshoot-Portfolio.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 240px;
+  width: 290px;
+}
```

Comment:
- [ ] Please update this so that your work will look professional. You can see the differences between your work section and the Figma one below. Comparing your work sections with the Figma design work section shows that they are slightly different. 😉  👇  

Your Design
![yoursection](https://user-images.githubusercontent.com/88894525/192094129-3fc20f33-838d-4041-af70-5d3a2a636cb1.png)

Figma Design 
![figmaSection](https://user-images.githubusercontent.com/88894525/192094155-3b531aa8-ceb4-4524-a416-5453472c6cac.png)

Reply:
- [ ] kindly make sure your button and their interaction `(hover, active, disable)` with the user look similar to the ones provided to you within the Figma template as required in the project.
`hint: you may need to change the border: 1px solid #<color-provided>` and `border-radius: 5px`